### ISSUE TITLE
Adjust honeybadger config to store the email address of a logged in OH requester in error context

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/scihist_digicoll#2579]
Useful for debugging

**Comment by eddierubeiz** [comment]
I'm looking at this after a few months, and I'm noting:

```ruby
before_action do
    timecode = Time.now.gmtime.to_i
    Honeybadger.context({
      :papertrail_url => "https://papertrailapp.com/events?focus=#{timecode}&selected=#{timecode}",
      :request_id => request.uuid,
      :current_user_email => current_user&.email
    })
  end
```

This is at: 
https://github.com/sciencehistory/scihist_digicoll/blob/9c52e9931dbf9028c8e05fcef05aed79142ebacc/app/controllers/application_controller.rb#L43

**Comment by eddierubeiz** [comment]
The email address of a logged in requester should indeed be the value of current_user&.email .

**Comment by jrochkind** [comment]
Are you saying it looks like we've already done it? makes sense to me. We could try to trigger somehow to verify

**Comment by eddierubeiz** [comment]
Exactly. Triggering an error in e.g. oral_history_requests_controller is an interesting idea, but I'm not going to do any more work on this issue this week.

**Comment by eddierubeiz** [comment]
Honestly I'd just rather wait until we have an HB error triggered by a known staff user (happens often enough).
Then we just need to go to the #notice-context tab on the error and ascertain that `current_user_email" => [non nil]

**Comment by jrochkind** [comment]
I feel like it only happens a couple times a year! And we need to remember this when it does, which we haven't before? But that's fine, whatever you think is best.

**Comment by eddierubeiz** [comment]
`current_user&.email` is `nil` in the controller we care about, which is `oral_history_requests_controller.rb`.
I'm going to add this value in manually.
Hopefully we never need to use this :)
